src/overview.xml

```diff
@@ -106,7 +106,7 @@ Traffic parameters |            | Statistical criteria
       bottlenecks would then be permitted to contribute a larger
       fraction of the end-to-end
       packet loss budget. In compensation,
-      subpaths that to not exhibit bottlenecks have must be constrained to contribute less packet
+      subpaths that to not expected exhibit bottlenecks must be constrained to contribute less packet
```

Comment:
Sentence still rough, maybe:
In compensation, subpaths that *are* not expected *to* exhibit bottlenecks must be constrained to contribute less packet loss.